gateways/http/api.go

```diff
@@ -39,7 +39,7 @@ func (api Api) Run(host string, port string) {
 	authenticatedRoute := r.PathPrefix("").Subrouter()
 	unauthenticatedRoute := r.PathPrefix("").Subrouter()
 	account.Accounts(unauthenticatedRoute, api.account)
-	transfer.Transfer(authenticatedRoute, api.transfer, api.account)
+	transfer.NewTransfer(authenticatedRoute, api.transfer, api.account)
```

Comment:
I believe something like `SetHandlers` would fit best, what do you think?

Reply:
This name fits for the func handler that is in account and transfer, but withdraw and deposit are in account too and what would their names look like? 😵‍💫

Reply:
maybe you could add the handler name to the function, as in `SetWithdrawHandler()` or something